src/app.js

```diff
@@ -0,0 +1,64 @@
+(() => {
+  const app = angular.module('Index', []);
+  app.controller('IndexCtrl', ($scope) => {
+    const newIndex = new InvertedIndex();
+    $scope.fileName = [];
+    $scope.fileCreated = [];
+    $scope.uploadedFiles = {};
+    $scope.upload = true;
+    $scope.myValue = false;
+    $scope.error = '';
+
+    $scope.documents = [];
+    $scope.details = {};
+    $scope.docNo = newIndex.docCount;
+
+    document.getElementById('myjsonfile').addEventListener('change', (e) => {
+      const fileDoc = e.target.files;
+      for (let x = 0; x < fileDoc.length; x += 1) {
```

Comment:
eslint doesn't allow this `x++`, I actually tried it before i resolved to use `x += 1` .